src/jdk.sctp/share/classes/com/sun/nio/sctp/ShutdownNotification.java

```diff
@@ -25,7 +25,7 @@
 package com.sun.nio.sctp;
 
 /**
- * Notification emitted when a peers shutdowns an the association.
+ * Notification emitted when a peers shutdowns the association.
```

Comment:
Maybe: "...when a peer shuts down an association."
(could be "shuts down the association" if there is only ever one, but using "an" looks safe)